schema/azure/defaults/infrastructure/cloud-os-image-defaults.yml

```
@@ -2,11 +2,11 @@ kind: infrastructure/cloud-os-image-defaults
 title: "Cloud OS Image Defaults"
 name: default
 specification:
-  ubuntu-18.04-x86_64:
+  ubuntu-20.04-x86_64:
     publisher: Canonical
-    offer: UbuntuServer
-    sku: 18.04-LTS
-    version: "18.04.202109130"
+    offer: 0001-com-ubuntu-server-focal
+    sku: 20_04-lts-gen2
+    version: "20.04.202112020"
```

```suggestion
    version: "20.04.202201040"
```
New images have just been published, let's use them :)

---

Image updated